packages/plugins/sentry/src/index.ts

```diff
@@ -220,29 +235,29 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
 
       return {
         onExecuteDone(payload) {
-          const handleResult: OnExecuteDoneHookResultOnNextHook<{}> = ({ result }) => {
+          const handleResult: OnExecuteDoneHookResultOnNextHook<{}> = ({ result, setResult }) => {
             if (includeRawResult) {
               rootSpan.setData('result', result);
             }
 
             if (result.errors && result.errors.length > 0) {
-              for (const err of result.errors) {
-                Sentry.withScope(scope => {
-                  scope.setTransactionName(opName);
-                  scope.setTag('operation', operationType);
-                  scope.setTag('operationName', opName);
-                  scope.setExtra('document', document);
+              Sentry.withScope(scope => {
+                scope.setTransactionName(opName);
+                scope.setTag('operation', operationType);
+                scope.setTag('operationName', opName);
+                scope.setExtra('document', document);
 
-                  scope.setTags(addedTags || {});
+                scope.setTags(addedTags || {});
 
-                  if (includeRawResult) {
-                    scope.setExtra('result', result);
-                  }
+                if (includeRawResult) {
+                  scope.setExtra('result', result);
```

Comment:
Side note:

Maybe we should consider using here `setContext`? 

This way the things we set now as `setExtra` would be displayed more nicely in Sentry UI